content/soarian/dstu2/diagnostic/observation.md

```diff
@@ -41,28 +41,28 @@ _Implementation Notes_
 
 * The API returns the most current charted Observation list. 
 
-* The Narrative includes additional information pertaining to the observation, and should be shown to the user to ensure completeness of clinical content and context
+* The API returns values in reverse chronological order by the most currently charted date.
 
-### Parameters 
-
-|Name |Required? | Type | Description
-| --- | --- | --- | --- |
-| patientId | Y | [reference](http://hl7.org/fhir/DSTU2/search.html#reference) | The patient identifier provided in a pre-requisite authorization step. |
-| date | N | [date](http://hl7.org/fhir/DSTU2/search.html#date) as adjusted per implementation notes | Null or specific date or a date range. |
-
-Notes:
+* The API returns charted content that is not marked as erroneous.
```

Comment:
Those aren't valid status values for Observation. Use the FHIR status values, developers won't be familiar with how Soarian maps to FHIR, the way you're stating it here, they would be required to understand Soarian mappings to FHIR and Soarian workflows.

http://hl7.org/fhir/dstu2/valueset-observation-status.html